apps/deploy-web/src/components/usage/UsageLayout.tsx

```
@@ -0,0 +1,54 @@
+  children?: ReactNode;
+};
+
+export const UsageLayout: React.FunctionComponent<Props> = ({ children, page }) => {
+  const router = useRouter();
+
+  const handleTabChange = (newValue: string) => {
+    switch (newValue as UsageTabs) {
+      case UsageTabs.BILLING:
+        router.push(UrlService.billing());
+        break;
+      case UsageTabs.USAGE:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)